docs/source/user-guide/supported_models.rst

```diff
@@ -0,0 +1,56 @@
+.. _supported_models:
```

Comment:
just a note - no need for a tag since documents can be directly referred to using a "doc" link... (anyway, no problem for adding either)